documentation/asciidoc/accessories/iqaudio/configuration.adoc

```diff
@@ -52,34 +56,33 @@ file will need to be written to the Pi’s SD card to allow the Pi to boot.
 
 === Attaching the HAT
 
-The IQaudio range of sound cards attach to the Raspberry Pi’s 2x20 way pin header (GPIO header).
-They are designed to be supported on the Pi using the supplied PCB standoffs and screws. In
-general no soldering is required to the IQaudio boards for normal operation.
+The Raspberry Pi range of sound cards attach to the Raspberry Pi’s 2x20 way pin header (GPIO header).
+They are designed to be supported on the Raspberry Pi using the supplied circuit board standoffs and screws. In
+general no soldering is required on the Raspberry Pi audio boards for normal operation with the exception when utilising hardwired connections for specific connectors, XLR (External Line Return) connections on the DAC Pro for example.
 
-The IQaudio range of boards are normally supplied with the necessary mounting hardware (PCB
-spacers and screws), these are for our latest low profile boards 4x 9mm PCB spacers and 8x M2.5
-screws. If additional parts (such as 2piece speaker connectors) are required these are normally
-provided too.
+The Raspberry Pi range of boards are normally supplied with the necessary mounting hardware (PCB
+spacers and screws), these are for our latest low profile boards 4 x 9mm PCB spacers and 8 x M2.5mm
+screws. If additional parts (such as 2 piece speaker connectors) are required these are normally
```

Comment:
"If additional parts" should be reworded, and "normally" should definitely be deleted.